packages/api/helper/mailer.js

```diff
@@ -83,11 +83,11 @@ function verifyHcEmail(nomination) {
       },
       locals: {
         name: nomination.providerName,
-        appUrl: process.env.APP_URL,
-        urlLink: `${process.env.APP_URL}/confirmation/${emailToken}`,
+        urlLink: `${process.env.APP_URL}/email-verification/${emailToken}`,
+        imgUrl,
       },
     })
-    .then(() => console.log('email has been sent!'))
+    .then(() => console.log('verify email has been sent!'))
```

Comment:
I don't think this console log is worth keeping. The user will likely not be looking at the console and you can always check the network tab of the debugging tools to make sure the API call went through.

Reply:
got it, will remove